CHANGELOG.md

```diff
@@ -74,10 +74,12 @@ Changes since v2.4
 - Email messages now use the application title and full names of users
 - Email message texts improved
 - Show three latest events as a default on the application page (#1632)
+- User's language setting is stored in a cookie so that it persists after login. Also the language is possibly correct already at login because the cookie is remembered.
```

Comment:
I don't understand the second sentence.

Reply:
I reworded these. Essentially the second sentence is that the language is possibly correct already at login because the language setting stored in the cookie is remembered :)